src/dao/V2FilterProcessor.ts

```diff
@@ -60,7 +61,11 @@ export class V2FilterProcessor implements IFilterProcessor {
       case Comparators.NOTNULL:
         return restriction.value === undefined ? V2FilterProcessor.NULL_VALUE : restriction.value;
```

Comment:
This is not related to the current issue, but: Originally the REST v2 endpoint required a `new Date(0)` for a `null` date. I am not sure if this is still the case, but if it is, we should consider it accordingly in the client code.